tsconfig.volar.json

```diff
@@ -3,5 +3,6 @@
   "compilerOptions": {
     "lib": ["DOM", "ES2020"],
     "skipLibCheck": true
-  }
+  },
+  "exclude": ["tests/expose.spec.ts"]
```

Comment:
volar can't figure out that we magically expose more things on `wrapper.vm`. That might be an issue long term as users may run into it as well.

Reply:
Seems like this isn't really something than can be fixed - it makes little sense for Volar to have something special for test utils.

Reply:
Yeah. I think there was an issue even with regular `defineExpose` (which should work), but we'll see if that's really an issue and what we can do about it.